### ISSUE TITLE
codelabs/firebase-ios-swift/ doesn't work with latest Xcode release

### ISSUE THREAD

**Issue by daviderb** [firebase/friendlychat-ios#62]
The Firebase iOS Swift codelab doesn't work with the current release of Xcode, as support for Swift 3 has been removed. Errors are mostly related to name changes.
To reproduce, pull the codelab and try to build it with Xcode 10.2.

**Comment by ulukaya** [comment]
Yes, this needs to be updated all both ios, ios-starter and grow-starter directories. And the respective codelab docs need to reflect the change.

**Comment by kamronthopkins** [comment]
> Yes, this needs to be updated all both ios, ios-starter and grow-starter directories. And the respective codelab docs need to reflect the change.

I agree, GOOGLE please fix these simple issues....

**Comment by tinalambert** [comment]
Has there been an update with this? I've been trying to get this set up for hours now...
